### ISSUE TITLE
test TLS certs expired

### ISSUE THREAD

**Issue by pravi** [abonas/kubeclient#483]
```
TestResourceQuota#test_get_from_json_v1 = 0.01 s = .
KubeclientConfigTest#test_user_token = /usr/lib/ruby/vendor_ruby/kubeclient/config.rb:33: Passing permitted_classes with the 2nd argument of Psych.safe_load is deprecated. Use keyword argument like Psych.safe_load(yaml, permitted_classes: ...) instead.
0.00 s = .
KubeclientConfigTest#test_external_nopath_absolute = 0.00 s = .
KubeclientConfigTest#test_gcp_default_auth = /tmp/autopkgtest-lxc.0kqlxqvn/downtmp/build.PLE/src/test/test_config.rb:128: Passing permitted_classes with the 2nd argument of Psych.safe_load is deprecated. Use keyword argument like Psych.safe_load(yaml, permitted_classes: ...) instead.
0.00 s = .
KubeclientConfigTest#test_gcp_default_auth_renew = /tmp/autopkgtest-lxc.0kqlxqvn/downtmp/build.PLE/src/test/test_config.rb:137: Passing permitted_classes with the 2nd argument of Psych.safe_load is deprecated. Use keyword argument like Psych.safe_load(yaml, permitted_classes: ...) instead.
0.00 s = .
KubeclientConfigTest#test_user_exec = /usr/lib/ruby/vendor_ruby/kubeclient/config.rb:33: Passing permitted_classes with the 2nd argument of Psych.safe_load is deprecated. Use keyword argument like Psych.safe_load(yaml, permitted_classes: ...) instead.
0.01 s = .
KubeclientConfigTest#test_user_password = /usr/lib/ruby/vendor_ruby/kubeclient/config.rb:33: Passing permitted_classes with the 2nd argument of Psych.safe_load is deprecated. Use keyword argument like Psych.safe_load(yaml, permitted_classes: ...) instead.
0.00 s = .
KubeclientConfigTest#test_user_exec_nopath = 0.00 s = .
KubeclientConfigTest#test_external = /usr/lib/ruby/vendor_ruby/kubeclient/config.rb:33: Passing permitted_classes with the 2nd argument of Psych.safe_load is deprecated. Use keyword argument like Psych.safe_load(yaml, permitted_classes: ...) instead.
0.00 s = F
KubeclientConfigTest#test_timestamps = /usr/lib/ruby/vendor_ruby/kubeclient/config.rb:33: Passing permitted_classes with the 2nd argument of Psych.safe_load is deprecated. Use keyword argument like Psych.safe_load(yaml, permitted_classes: ...) instead.
0.00 s = .
KubeclientConfigTest#test_allinone = /usr/lib/ruby/vendor_ruby/kubeclient/config.rb:33: Passing permitted_classes with the 2nd argument of Psych.safe_load is deprecated. Use keyword argument like Psych.safe_load(yaml, permitted_classes: ...) instead.
0.00 s = F
KubeclientConfigTest#test_allinone_nopath = 0.00 s = F
KubeclientConfigTest#test_nouser = /usr/lib/ruby/vendor_ruby/kubeclient/config.rb:33: Passing permitted_classes with the 2nd argument of Psych.safe_load is deprecated. Use keyword argument like Psych.safe_load(yaml, permitted_classes: ...) instead.
0.00 s = .
```

Finished in 0.631563s, 240.6727 runs/s, 932.6065 assertions/s.

  1) Failure:
KubeclientConfigTest#test_external [/tmp/autopkgtest-lxc.0kqlxqvn/downtmp/build.PLE/src/test/test_config.rb:199]:
Expected false to be truthy.

  2) Failure:
KubeclientConfigTest#test_allinone [/tmp/autopkgtest-lxc.0kqlxqvn/downtmp/build.PLE/src/test/test_config.rb:199]:
Expected false to be truthy.

  3) Failure:
KubeclientConfigTest#test_allinone_nopath [/tmp/autopkgtest-lxc.0kqlxqvn/downtmp/build.PLE/src/test/test_config.rb:199]:
Expected false to be truthy.

Full build log https://ci.debian.net/data/autopkgtest/testing/amd64/r/ruby-kubeclient/8232365/log.gz

This was working earlier, but recently started failing. Packages that changed between the success and failure is.

-openssl	1.1.1g-1
+openssl	1.1.1h-1
-ruby-bindata	2.3.5-1
+ruby-bindata	2.4.8-1

**Comment by pravi** [comment]
The same failure is in master as well https://travis-ci.org/github/abonas/kubeclient/jobs/743368719

**Comment by pravi** [comment]
With tests passing earlier on travis with bindata 2.4.8, I think it is down to openssl update.

**Comment by cben** [comment]
The certificate used in these tests expired.
We temporarily disabled these tests on master, and I'm working on replacing the cert...

**Comment by cben** [comment]
Renamed issue for clarity.

The "Passing permitted_classes with the 2nd argument of Psych.safe_load is deprecated" message was just a warning, fixed by #469. (These warning partially prepared us for ruby 3.0 compatibility, but not fully — WIP on #489)

**Comment by cben** [comment]
Not certain what's the value of the current test.
But one thing that _would_ be valuable to test is https://github.com/abonas/kubeclient/pull/461
